src/NHibernate/Type/TimestampUtcType.cs

```diff
@@ -1,70 +0,0 @@
-﻿using System;
-using System.Data.Common;
-using NHibernate.Engine;
-
-namespace NHibernate.Type
-{
-	[Serializable]
-	public class TimestampUtcType : TimestampType
```

Comment:
Need to remove from mapping schema

Reply:
Had not thought it was even added as a specif versioning element shortcut for `<version type="timestamputc"`

Reply:
Maybe remove it in a separate commit?

Reply:
Remove the whole timestamputc stuff with a commit like "[NH-2520](https://nhibernate.jira.com/browse/NH-2520) - Revert TimestampUtc in favor of UtcDateTime"

Reply:
yes